resources/aws/bucket.go

```diff
@@ -8,13 +8,18 @@ import (
 	"github.com/aws/aws-sdk-go/service/s3"
 	microerror "github.com/giantswarm/microkit/error"
 	"github.com/juju/errgo"
+
+	"github.com/giantswarm/aws-operator/resources"
 )
 
 type Bucket struct {
 	Name string
 	AWSEntity
 }
 
+// Implement ReusableResource.
+var _ = resources.ReusableResource(&Bucket{})
```

Comment:
Why do we need to ensure this here? It should be ensured where it is actually being used. So this should be good enough.